client/src/commands/encointer_core.rs

```diff
@@ -44,11 +44,14 @@ pub fn balance(_args: &str, matches: &ArgMatches<'_>) -> Result<(), clap::Error>
 				println! {"{balance:?}"};
 			},
 			None => {
+				if maybe_at.is_some() {
+					panic!("can't apply --at if --cid not set")
+				};
```

Comment:
It will be tricky to support `--at` here because we make heavy use of custom rpc's